to.etc.domui/src/main/resources/resources/themes/css-domui/670_wizard.frag.css

```diff
@@ -0,0 +1,34 @@
+/** Generic Wizard Component **/
+div.wizard-navbar {
+    position: absolute;
+    width: 100%;
+    top: 28px;
+    left: 0;
+    display: flex;
+}
+
+div.wizard-body {
+    margin-top: 40px;
```

Comment:
I assume that you want to make a fixed top bar with a flex container for the step body.
To do that you make the top container position as follows:
top: 0px
bottom: 28px
left: 0px
right: 0px
position: absolute
In your code here the top bar behaves oddly: the top is 28 pixels below the dialog start, and it can grow vertically if the line becomes too long. So the div does not overlay the full parent part.

The container for the step should be:
position: absolute
top: 28px
left: 0px
right: 0px
bottom: 0px

This makes the bottom part resize fully, with the top part being exactly 28 pixels high and overlaying the full parent area.